### PR TITLE
i18n but secret

### PR DESCRIPTION
**Pull request by nonowazu**
This just adds i18n capabilities and just ~~infects~~ changes one file to use it for the sake of example. Right now we only support English, but if we support more, it should be easy to add in (just edit the i18n config file and locale and run `yarn extract`)

I do expect some harsh scrutiny on this one, but I just think it will make for a better PR overall with that level of examination. Areas that probably need work:

1. perhaps better tooling with `yarn extract`
2. some typing around the json file(s) so imports with them look a little cleaner
3. Some documentation in the locale folder with suggestions about key hierarchies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,12 +11,14 @@
     "lint": "yarn lint:eslint && yarn lint:prettier",
     "lintfix": "eslint . --fix && prettier --write --list-different .",
     "postinstall": "nuxt prepare",
-    "preview": "nuxt preview"
+    "preview": "nuxt preview",
+    "extract": "vue-i18n-extract report --vueFiles './{pages,components}/**/*.vue' --languageFiles './locale/*.json' --add"
```
Comment (dpcodes) [inline package.json:15]: what does this do, and do we need to add it to CI

Comment (dpcodes) [reply package.json:15]: nvm found it online rofl, that's really good actually

**Comment by dpcodes** [comment]
Have you tried creating a second resource file for another locale to see if it picks it up when you try another language?

File: components/RoleSlider.vue [hunk 2]
```diff
@@ -144,9 +144,9 @@ export default {
       swiperNav.value?.slideTo(swiper.activeIndex);
     }
 
-    const roles: Role[] = data.value?.roles.toSorted(
+    const roles: Role[] = data.value?.roles?.toSorted(
       (a: Role, b: Role) => a.order - b.order,
-    );
+    ) ?? [];
```
Comment (nonowazu) [inline components/RoleSlider.vue:149]: This is a mistake and I'll fix it on the next pass.

**Review by dpcodes — APPROVED** [review]
This is good to me, just need to address the CI.

**Pull request closed without merging**